src/Materials/babylon.uniformBuffer.ts

```diff
@@ -0,0 +1,584 @@
+module BABYLON {
+
+    // Pool for avoiding memory leaks
+    var MAX_UNIFORM_SIZE = 256;

```

Comment:
should not be on BABYLON namespace but more as a private const inside the buffer